db/dummy_seed.rb

```diff
@@ -0,0 +1,248 @@
+  },
+
+  {
+    first_name: "Antonio",
+    last_name: "Seaman",
+    nickname: "Notable Pirate",
+    landline: "",
+    cell: "",
+    email: "sailorboy44@piratebay.com",
+    address: "",
+    neighborhood: "Twelfth Night",
+    spouse: "",
+    biography: "Friend to Sebastian.  Probably gay.",
+  },
+  {
+    first_name: "Sebastian",
+    last_name: "Of Messaline",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "sebastian@messaline.gov",
+    address: "",
+    neighborhood: "Twelfth Night",
+    spouse: "",
+    biography: "Supporting romantic lead.",
+  },
+  {
+    first_name: "Viola",
+    last_name: "Of Messaline",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "viola@messaline.gov",
+    address: "",
+    neighborhood: "Twelfth Night",
+    spouse: "",
+    biography: "The shipwrecked protagonist.",
+  },
+  {
+    first_name: "Orsino",
+    last_name: "Of Illyria",
+    nickname: "The Dewke",
+    landline: "",
+    cell: "",
+    email: "orsino@illyria.gov",
+    address: "",
+    neighborhood: "Twelfth Night",
+    spouse: "",
+    biography: "I must be the sexiest man on stage.",
+  },
+
+  {
+    first_name: "Alexander",
+    last_name: "Hamilton",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "Aaron",
+    last_name: "Burr",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "Thomas",
+    last_name: "Jefferson",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "Anjelica",
+    last_name: "Schuyler",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "Eliza",
+    last_name: "Schuyler",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "Peggy",
+    last_name: "Schuyler",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "George",
+    last_name: "Washington",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+  {
+    first_name: "James",
+    last_name: "Maddison",
+    nickname: "",
+    landline: "",
+    cell: "",
+    email: "",
+    address: "",
+    neighborhood: "Hamilton",
+    spouse: "",
+    biography: "",
+  },
+]
+
+profiles.each do |profile|
+  Profile.create! profile
+end
```

Comment:
Do we have a dummy seed for users as well? Given the work we're doing over there, I feel like it would be useful.

Reply:
I just created #169  so that that can be a separate/new issue on its own. Lets keep branches/issues/features as atomic as possible.